src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -2202,35 +2210,35 @@ class ObservationStore(
               null
             }
 
+        val survivalRateDenominator =
+            DSL.field(
+                DSL.select(DSL.sum(PLOT_T0_DENSITY.PLOT_DENSITY))
+                    .from(PLOT_T0_DENSITY)
+                    .where(survivalRateDensityCondition)
+                    .and(PLOT_T0_DENSITY.SPECIES_ID.eq(speciesKey.id))
+            )
+        val survivalRate =
+            if (isPermanent && speciesKey.id != null) {
+              DSL.`val`(totalLive).mul(100).div(survivalRateDenominator)
```

Comment:
`DSL.value` is a synonym for `DSL.val` but the name is more Kotlin-friendly.

Reply:
Ah I did see that after using `val`. I saw that we use both, but I should have paid better attention to notice that we almost always use `value`. Will update